resources/CICD/GithubActions/deploy_Release_Firebase.yml

```diff
@@ -0,0 +1,73 @@
+name: deploy_Release_Firebase
```

Comment:
Please update the file name and the name here
- `deploy_release_firebase.yml`
- `Deploy Release Build To Firebase`